data/free worlds start.txt

```diff
@@ -1418,6 +1531,70 @@ mission "FW Pirates 3.2B"
+
+
+
+mission "FW Pirates 3.1D"
+	name "Get the Freed Slaves to Safety"
+	description "Escort a fleet of freed slaves to Longjump and answer for your decision."
+	landing
+	to offer
+        has "FW Pirates 3.1C: active"
+	destination Longjump
+	on fail
+        dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	on offer
+	set "fw pirates: victory"
+	conversation
+	    	`You bring your ship to hover over the spaceport and announce that they have one hour to release their captives and leave the area on the ground. Cowed by your guns, the slave market complies. The few remaining pirates and local thugs throw their valuables into any trucks they can find and head for the hills.`
+		`	Some of the freed slaves are pilots, technicians, and engineers, and they commandeer a dozen assorted transports and freighters. You spend half a day helping them liberate slave pens and labor camps around the planet. Under the threat of your guns, the slaves help themselves to the property of their former masters, and the transports and freighters are crammed with enough wealth, equipment, and weapons to enable a fresh start on a new world. Slave-masters who don't comply lose their farms, factories, and in many cases, their lives.`
+		`	Then you turn your guns onto every piece of infrastructure you can find that supports the pirate fleet. After several hours, the pirate's shipyard, outfitter, and refueling stations are destroyed beyond any hope of repair. Tracing the ion trails of other ships that launched to attack you, you find and destroy their bases of operaion, too. Former slaves are quick to point out key infrastructure that would otherwise have escaped your notice. It will take Greenrock decades to rebuild their pirate fleet and support structure, and they'll have to rebuild it without slave labor.`
+		`	Now you must get this rag-tag fleet full of escaped slaves to safety and answer for your actions.`
+		launch
+	npc accompany save
+		government Merchant
+		personality escort timid
+		ship "Behemoth" "Emancipation"
+		ship "Behemoth" "Freedom Fry"
+		ship "Behemoth" "Hope"
+		ship "Bulk Freighter" "Bondage Breaker"
+		ship "Bulk Freighter" "Tubman"
+		ship "Bulk Freighter" "Liberty"
+		ship "Hauler" "Moses"
+		ship "Hauler II" "Exodus"
+		ship "Hauler II" "Epaminondas"
+		ship "Hauler III" "Lincoln"
+		ship "Star Queen" "Shattered Shackle"
+		ship "Star Queen" "Sherman"
+		ship "Star Queen" "New Life"
+		ship "Star Queen" "Escape"
+		ship "Argosy" "Broken Chain"
+		ship "Argosy" "Repatriation"
+		ship "Freighter" "Deliverance"
+		ship "Freighter" "Amnesty"
+	on complete
+		payment 120000
+		dialog "The captain of the <npc> thanks you for escorting them safely, and pays you <payment>."
```

Comment:
I think you wan't to remove the last 2 lines in favor of having the next bit of story in the next mission.